contracts/Insurance.sol

```diff
@@ -99,21 +100,26 @@ contract Insurance is IInsurance, Ownable, SafetyWithdraw {
         uint256 wadTokensToSend =
             LibInsurance.calcWithdrawAmount(
```

Comment:
Just confiming that this function only allows you to withdraw from publc (collateralAmount), and not from bufferAmount.

Reply:
Yeah, I think `calcWithdrawAmount` should only take public collat amount, as opposed to public + buffer, right?

Reply:
Ooh, you're definitely right; fixed now